### PR TITLE
Make joy-types package into a reusable standalone npm package

### PR DESCRIPTION
**Pull request by mnaamani**
Published the package at https://www.npmjs.com/package/@joystream/types

- [x] Add missing keys in package.json to show better on npmjs
- [x] Add Readme and show example how to use package

### CODE REVIEW

**Comment by mnaamani** [comment]
Example usage https://github.com/Joystream/joystream-api-examples

File: packages/joy-types/README.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+# `@joystream/types`
+
+The types packages is required to register the custom substrate runtime types when working with [`@polkadot/api`](https://www.npmjs.com/package/@polkadot/api#registering-custom-types) to communicate with a joystream full node.
```
Comment (siman) [inline packages/joy-types/README.md]:
```suggestion
The types package is required to register the custom Substrate runtime types when working with [`@polkadot/api`](https://www.npmjs.com/package/@polkadot/api#registering-custom-types) to communicate with a Joystream full node.
```

File: packages/joy-types/README.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Installation
+
+Add the package as a dependency in your project.
+
+```shell
+yarn add @joystream/types
+
+or
```
Comment (siman) [inline packages/joy-types/README.md]:
```suggestion
# or
```

File: packages/joy-types/README.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+npm install --save @joystream/types
+```
+
+## Registering the types
+
+Call `registerJoystreamTypes()` before creating a polkadot api client.
```
Comment (siman) [inline packages/joy-types/README.md]:
```suggestion
Call `registerJoystreamTypes()` before creating a Polkadot API client.
```

File: packages/joy-types/README.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+  // Initialise the provider to connect to the local node
+  const provider = new WsProvider('ws://127.0.0.1:9944');
+
+  // Register types before creating the API
+  registerJoystreamTypes();
+
+  // Create the API and wait until ready
+  const api = await ApiPromise.create(provider);
+
+  // Retrieve the chain & node information information via rpc calls
```
Comment (siman) [inline packages/joy-types/README.md]:
```suggestion
  // Retrieve the chain & node information information via RPC calls
```

File: packages/joy-types/package.json [hunk 5]
```diff
@@ -1,14 +1,37 @@
...
+  "keywords": [
+    "substrate",
+    "joystream",
+    "runtime"
+  ],
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/Joystream/apps/issues"
+  },
+  "homepage": "https://github.com/Joystream/apps#readme"
```
Comment (siman) [inline packages/joy-types/package.json]:
```suggestion
  "homepage": "https://github.com/Joystream/packages/joy-types/README.md"
```

**Pull request merged**